app/controllers/invites_controller.rb

```diff
@@ -0,0 +1,25 @@
+require "http"
+
+class InvitesController < ApplicationController
+  skip_before_action :authenticate, only: []
+  skip_before_action :check_admin, only: []
```

Comment:
If we were to remove something, wouldn't we remove the two lines completely? I think we intend for only admins to be able to send invites.